torch/_torch_docs.py

```diff
@@ -8900,18 +8900,23 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.argsort,
            r"""
-argsort(input, dim=-1, descending=False) -> LongTensor
+argsort(input, *, dim=-1, descending=False, stable=True) -> Tensor
 
 Returns the indices that sort a tensor along a given dimension in ascending
 order by value.
 
 This is the second value returned by :meth:`torch.sort`.  See its documentation
 for the exact semantics of this method.
 
+If :attr:`stable` is ``True`` then the sorting routine becomes stable, preserving
+the order of equivalent elements. If ``False``, the relative order of values
+which compare equal is not guaranteed.
+
 Args:
     {input}
     dim (int, optional): the dimension to sort along
     descending (bool, optional): controls the sorting order (ascending or descending)
+    stable (bool, optional): controls the relative order of equivalent elements
```

Comment:
Could you please update examples to show this kwarg in action?

Reply:
I also think we should point out that stable sorting is slower, but preserves the relative order of equivalent elements...

Reply:
I added some `doctest`s, but let me know if these would be better in the unit tests. I've also updated the documentation.

Reply:
Doctests are examples, not tests. You always should add a regular unit test for adding a new keyword.

Reply:
I second to that. Let's add/update tests as well. You cannot imagine how many bugs were missed to such a trivial matter :)